src/main/java/httpserver/http/ProtocolVersion.java

```diff
@@ -0,0 +1,15 @@
+package httpserver.http;
+
+public enum ProtocolVersion {
```

Comment:
Nice - although I think this is the full protocol, not just the protocol version.

Reply:
Thank you, I'll change this